metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -94,8 +94,8 @@ case class ScalafixProvider(
               exception
             )
             Future.failed(exception)
-          case Success(results) if !results.isSuccessful =>
-            val scalafixError = results.getErrorMessage().asScala
+          case Success(results) if !scalafixSucceded(results) =>
+            val scalafixError = getMessageErrorFromScalafix(results)
             val message = scalafixError.getOrElse(defaultErrorMessage)
```

Comment:
Is this default actually still needed? Will `scalafixError` actually ever be `None` here? If I'm understanding the pr description correctly one of the possible errors that a file evaluation would fail would be 

> StaleSemanticdbError, MissingSemanticdbError.

And this is more or less what `defaultErrorMessage` is. So since we get the error messages for each file down below in `getMessageErrorFromScalafix` is this default still needed?

Reply:
No the defaut il not needed anymore.